<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuthWindow</class>
 <widget class="QDialog" name="AuthWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Авторизация</string>
  </property>
  <property name="windowIcon">
   <iconset resource="IMG.qrc">
    <normaloff>:/IMG/IMG/logoSN.png</normaloff>:/IMG/IMG/logoSN.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(248, 249, 255);</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>390</y>
     <width>341</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: black;</string>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="textLogin">
   <property name="geometry">
    <rect>
     <x>79</x>
     <y>100</y>
     <width>341</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
     <stylestrategy>PreferDefault</stylestrategy>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
    background-color: white;
    color: black;              
    border-radius: 20px;      
    padding: 8px 12px;        
    border: 1px solid black; 
}</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>25</number>
   </property>
   <property name="placeholderText">
    <string>Логин</string>
   </property>
  </widget>
  <widget class="QLabel" name="authLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>25</y>
     <width>220</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>20</pointsize>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text">
    <string>Авторизация</string>
   </property>
  </widget>
  <widget class="QPushButton" name="passwordLink">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>400</y>
     <width>131</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton 
{
    background: transparent;
    color: #8A2BE2;
    border: none;
    text-align: left;
    padding: 0;
}
QPushButton:hover { color: #9932CC; text-decoration: underline; }</string>
   </property>
   <property name="text">
    <string>забыли пароль</string>
   </property>
  </widget>
  <widget class="QPushButton" name="authButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>310</y>
     <width>341</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton 
{
    background: qlineargradient(
        x1:0, y1:0, x2:0, y2:1,
        stop:0 #6a3093, 
        stop:1 #a044ff 
    );
    color: white;     
    border-radius: 20px;
    padding: 8px 16px;  
    font-weight: bold;   
    border: 1px solid #4a148c; 
}

QPushButton:hover
 {
    background: qlineargradient(
        x1:0, y1:0, x2:0, y2:1,
        stop:0 #7b1fa2,
        stop:1 #aa00ff
    );
    border: 2px solid #7b1fa2;
}

QPushButton:pressed 
{
    background: qlineargradient(
        x1:0, y1:0, x2:0, y2:1,
        stop:0 #4a148c,
        stop:1 #7b1fa2
    );
}</string>
   </property>
   <property name="text">
    <string>Войти</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="textPassword">
   <property name="geometry">
    <rect>
     <x>79</x>
     <y>170</y>
     <width>341</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
    background-color: white;
    color: black;              
    border-radius: 20px;      
    padding: 8px 12px;        
    border: 1px solid black; 
}</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="maxLength">
    <number>25</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::EchoMode::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Пароль</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeEye">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>175</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton 
{
    background: transparent;
    border: none;
    text-align: left;
    padding: 0;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="IMG.qrc">
     <normaloff>:/IMG/IMG/CloseEye35x35SN.png</normaloff>:/IMG/IMG/CloseEye35x35SN.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>35</width>
     <height>35</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="regLink">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>400</y>
     <width>111</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton 
{
    background: transparent;
    color: #8A2BE2;
    border: none;
    text-align: left;
    padding: 0;
}
QPushButton:hover { color: #9932CC; text-decoration: underline; }</string>
   </property>
   <property name="text">
    <string>регистрация</string>
   </property>
  </widget>
  <widget class="QPushButton" name="openEye">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>175</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton 
{
    background: transparent;
    border: none;
    text-align: left;
    padding: 0;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="IMG.qrc">
     <normaloff>:/IMG/IMG/OpenEye35x35SN.png</normaloff>:/IMG/IMG/OpenEye35x35SN.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>35</width>
     <height>35</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="textPhone">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>240</y>
     <width>341</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
    background-color: white;
    color: black;              
    border-radius: 20px;      
    padding: 8px 12px;        
    border: 1px solid black; 
}</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="maxLength">
    <number>25</number>
   </property>
   <property name="placeholderText">
    <string>Телефон</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="IMG.qrc"/>
 </resources>
 <connections/>
</ui>
